e2e/provision/README.md

```diff
@@ -0,0 +1,49 @@
+# Quick Start for GCE
+
+1. Create a VM:
+
+```bash
+$ gcloud compute instances create --machine-type e2-standard-8 --boot-disk-size 60GB nephio-r1-e2e
+```
+
+2. SSH to the VM:
+
+Googlers (you also need to run `gcert`):
+```bash
+$ gcloud compute ssh nephio-r1-e2e -- -o ProxyCommand='corp-ssh-helper %h %p' -L 7007:localhost:7007
+```
+
+Everyone else:
+```bash
+$ gcloud compute ssh nephio-r1-e2e -- -L 7007:localhost:7007
+```
+
+3. On the machine, run this:
+
+```bash
+$ sudo apt-get install -y git
+$ git clone https://github.com/nephio-project/test-infra.git
+$ sed -e "s/vagrant/$USER/" < test-infra/e2e/provision/nephio.yaml > ~/nephio.yaml
+$ cd test-infra/e2e/provision/
+$ ./gce_run.sh
+$ kubectl --kubeconfig ~/.kube/mgmt-config port-forward --namespace=nephio-webui svc/nephio-webui 7007
```

Comment:
I think that this needs to be executed with `sudo`
```suggestion
$ sudo kubectl --kubeconfig ~/.kube/mgmt-config port-forward --namespace=nephio-webui svc/nephio-webui 7007
```

Reply:
No, this works fine without sudo since it's not a privileged port.

Reply:
I know what happen, the `one-summit` repo is used by [default](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/gce_run.sh#L48). The CI has been changed to use the `cluster-api` with the `DEPLOYMENT_TYPE` env var.